rcv/nes56_cv_manager.py

```diff
@@ -0,0 +1,150 @@
+import constants
+import cv2
+import logging
+import os
+import rcv_utils
+import socket
+import sys
+
+
+
+# globals
+API_structure = {
+        "front": False,
+        "found": False,
+        "p1": {"d": 0, "a": 0},
+        "p2": {"d": 0, "a": 0},
+    }
+
+
+class Nes56CvManager():
+    def __init__(self, conf):
+        logging.debug("entered Nes56CvManager.__init__(...)")
+        self._stop = False
+        self._conf = conf
+        self._client_socket = None
+        self._cap = None
+        self._width = None
+        self._height = None
+        self._video_source_type = None
+        self._video_source = None
+        self._camera_name = None
+        self._fps = None
+        self._roborio_ip = self._conf.get('rcv_server', 'roborio_ip')
+        self._roborio_port = int(self._conf.get('rcv_server', 'roborio_port'))
+        self._socket_timeout = int(self._conf.get('rcv_server', 'socket_timeout'))
+        self._show = bool(self._conf.get('rcv_server', 'show'))
+        self._camera_height=float(self._conf.get('rcv_server', 'camera_height'))
+        self._y_leaning_angle=float(self._conf.get('rcv_server', 'y_leaning_angle'))
+        self._x_turning_angle=float(self._conf.get('rcv_server', 'x_turning_angle'))
```

Comment:
I think that the camera height y leaning angle and x turning angle shouldn't be stored here but only in the frc_2019.py because it isn't needed here and if we change our logic we won't need these

Reply:
I disagree,
These configurations belong to the camera and not to the frame,
And this is the class that handles the camera specific configuration.
Although I have stored them also in the frc_2019 module, basically these configurations belong here more than there.
However, I do not have a strong opinion about it.
You can change it if you wish once it is merged